src/transport/RendezvousSession.cpp

```diff
@@ -83,6 +153,15 @@ void RendezvousSession::OnPairingError(CHIP_ERROR err)
 void RendezvousSession::OnPairingComplete()
 {
     mPairingInProgress = false;
+
+    CHIP_ERROR err = mPairingSession.DeriveSecureSession((const unsigned char *) kSpake2pI2RSessionInfo,
+                                                         strlen(kSpake2pI2RSessionInfo), mSecureSession);
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(Ble, "Failed to initialize a secure session: %s", ErrorStr(err));
+        return;
+    }
+
```

Comment:
Any reason not to follow suit and use the `SuccessOrExit(err);` and `exit:` design here?

Reply:
Done.